cmd/edgectl/aes_install.go

```diff
@@ -453,6 +458,18 @@ func (i *Installer) Perform(kcontext string) error {
 		return err
 	}
 
+	versions, err := i.CaptureKubectl("get versions", false, "", "version", "-o", "json")
```

Comment:
@ark3 I'm extracting kubectl and the remote cluster version here. We should reuse `i.k8sVersion` for Metriton. (another PR)